docs/how-tos/using-pynxtools-test-framework.md

```diff
@@ -0,0 +1,56 @@
+# Generalized Test Functionality for `pynxtools` plugin
+The `pynxtools` sub-package `testing` is to be utilized to write automated test for `pynxtools` reader plugins without having any deeper knowledge of the `pynxtools` internal architecture. The tool mainly supports generalised test for all the reader plugins disregarding the technical details of raw file and internal design of the plugin (It is assumed that the plugin has followed the [plugin template](https://github.com/FAIRmat-NFDI/pynxtools-plugin-template)).
```

Comment:
```suggestion
# Generalized Test Functionality for `pynxtools` plugins
```suggestion
The `pynxtools` sub-package `testing` is to be utilized to write automated tests for `pynxtools` reader plugins without requiring in-depth knowledge of the `pynxtools` internal architecture. The tool supports generalised a general test for all reader plugins, irrespective of the technical details of the raw data files and the internal design of the plugin (note: it is assumed that the plugin was built from the [plugin template](https://github.com/FAIRmat-NFDI/pynxtools-plugin-template) or has the same structure internally).
```
```

Reply:
Noted!